test/basic-test/run

```diff
@@ -5,9 +5,11 @@
 
 base="$(dirname $0)"
 
+"$base/setup" "$@"
```

Comment:
This separate step is pain to use, and not needed since the drpolicy is used by one application.